libcontainer/process_linux.go

```diff
@@ -271,6 +296,30 @@ func (p *initProcess) createNetworkInterfaces() error {
 	return nil
 }
 
+func runCmd(command configs.Command, statePath string) error {
+	cmd := exec.Cmd{
+		Path: command.Path,
+		Args: command.Args[:],
+	}
+	cmd.Env = command.Env
+	cmd.Dir = command.Dir
```

Comment:
Might as well set these in the `cmd`-initializing line, now that we're going straight from the struct.